core/src/main/resources/hudson/tasks/Fingerprinter/help.html

```diff
@@ -5,19 +5,14 @@
 
   <ul>
     <li>
-      I have <tt>foo.jar</tt> on my HDD but which build number of FOO did it come from?
+      I want to know where a specific <code>foo.jar</code> came from.
```

Comment:
```suggestion
      I want to know <a href="${rootURL}/fingerprintCheck">where</a> a specific <code>foo.jar</code> came from.
```

Reply:
No Jelly variables like `rootURL` in HTML files.

Reply:
Sigh. Is the path guaranteed so we can use `..` the right number of levels?

Reply:
This gets inserted in job configuration forms, so no -- some might be in folders.

IIRC help files can also be Jelly if served via `Descriptor#doHelp`, so a file rename should do it, see e.g. https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/tasks/Maven/help.jelly

Please test it locally first though.

Reply:
good to know 👍